src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -364,35 +364,28 @@ def index_aip_and_files(
 
     :param client: The ElasticSearch client.
     :param uuid: The UUID of the AIP we're indexing.
-    :param path: path on disk where the AIP is located.
-    :param path: path on disk where the AIP's METS file is located.
+    :param aip_stored_path: path on disk where the AIP is located.
+    :param mets_staging_path: path on disk where the AIP METS file is located.
     :param name: AIP name.
-    :param size: optional AIP size.
+    :param aip_size: AIP size.
     :param aips_in_aic: optional number of AIPs stored in AIC.
     :param identifiers: optional additional identifiers (MODS, Islandora, etc.).
     :param identifiers: optional AIP encrypted boolean (defaults to `False`).
     :param printfn: optional print funtion.
     :return: 0 is succeded, 1 otherwise.
     """
-    # Stop if AIP or METS file don't not exist
+    # Stop if AIP or METS file don't not exist.
```

Comment:
Is this comment still correct? I don’t see a check for the existence of the AIP file.

Reply:
Your're right. I missed that comment. I'll remove it before merging. Thanks @alexwlchan.

Reply:
Fix it rather. The METS check is still there.